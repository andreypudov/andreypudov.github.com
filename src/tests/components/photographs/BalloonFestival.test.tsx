import { MemoryRouter } from 'react-router-dom'
import Component from './../../../components/photographs/BalloonFestival';
import { HelmetProvider } from 'react-helmet-async';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <HelmetProvider>
        <MemoryRouter>
          <Component />
        </MemoryRouter>
      </HelmetProvider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});