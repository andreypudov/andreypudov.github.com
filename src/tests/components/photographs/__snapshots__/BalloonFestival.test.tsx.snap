// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="layout"
>
  <header>
    <div
      className="container"
    >
      <div
        className="navbar navbar-expand-md navbar-dark"
      >
        <div
          className="container-fluid"
        >
          <a
            className="navbar-brand mx-auto"
            href="/en/"
            onClick={[Function]}
          >
            Header.AndreyPudov
          </a>
          <button
            aria-controls="navbarMain"
            aria-expanded="false"
            aria-label="Toggle navigation"
            className="navbar-toggler"
            data-bs-target="#navbarMain"
            data-bs-toggle="collapse"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </button>
          <div
            className="collapse navbar-collapse justify-content-sm-center"
            id="navbarMain"
          >
            <ul
              className="navbar-nav"
            >
              <li
                className="nav-item"
              >
                <a
                  className="nav-link"
                  href="/en/albums/"
                  onClick={[Function]}
                >
                  Header.Albums
                </a>
              </li>
              <li
                className="nav-item"
              >
                <a
                  className="nav-link"
                  href="/en/contents/"
                  onClick={[Function]}
                >
                  Header.Contents
                </a>
              </li>
              <li
                className="nav-item"
              >
                <a
                  className="nav-link"
                  href="/en/resume/"
                  onClick={[Function]}
                >
                  Header.Resume
                </a>
              </li>
              <li
                className="nav-item"
              >
                <a
                  className="nav-link"
                  href="/en/contact/"
                  onClick={[Function]}
                >
                  Header.Contact
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </header>
  <main>
    <div
      className="container"
    >
      <h1>
        Balloon Festival
        <small
          className="text-muted"
        >
          Saturday, February 24, 2018
        </small>
      </h1>
      <figure>
        <img
          alt="Balloon Festival"
          className="img-responsive gallery-image"
          src="/photographs/2018-02-24/2018-02-24_001.webp"
          title="Balloon Festival"
        />
      </figure>
      <figure>
        <img
          alt="Balloon Festival"
          className="img-responsive gallery-image"
          src="/photographs/2018-02-24/2018-02-24_002.webp"
          title="Balloon Festival"
        />
      </figure>
      <figure>
        <img
          alt="Balloon Festival"
          className="img-responsive gallery-image"
          src="/photographs/2018-02-24/2018-02-24_003.webp"
          title="Balloon Festival"
        />
      </figure>
      <figure>
        <img
          alt="Balloon Festival"
          className="img-responsive gallery-image"
          src="/photographs/2018-02-24/2018-02-24_004.webp"
          title="Balloon Festival"
        />
      </figure>
    </div>
  </main>
  <footer>
    <div
      className="container"
    >
      <hr
        className="footer-divider"
      />
      <ul
        className="piped-links float-end"
      >
        <li
          className="first piped-links-short"
        >
          <a
            href="/en/resume/"
            onClick={[Function]}
          >
            Footer.AboutShort
          </a>
        </li>
        <li
          className="first piped-links-long"
        >
          <a
            href="/en/resume/"
            onClick={[Function]}
          >
            Footer.AboutLong
          </a>
        </li>
        <li>
          <a
            href="/en/contact/"
            onClick={[Function]}
          >
            Footer.Contact
          </a>
        </li>
      </ul>
      <ul
        className="piped-links"
      >
        <p
          className="piped-links-long"
        >
          Footer.CopyrightLong
        </p>
        <p
          className="piped-links-short"
        >
          Footer.CopyrightShort
        </p>
      </ul>
    </div>
  </footer>
</div>
`;
